query-node/schema.graphql

```diff
@@ -12,30 +12,142 @@ type Block @entity {
   network: Network!
 }
 
+enum MembershipEntryMethod {
+  PAID
+  SCREENING
+  GENESIS
+}
+
 "Stored information about a registered user"
-type Member @entity {
+type Membership @entity {
   "MemberId: runtime identifier for a user"
   id: ID!
 
   "The unique handle chosen by member"
-  handle: String @unique @fulltext(query: "membersByHandle")
+  handle: String! @unique @fulltext(query: "membersByHandle")
 
   "A Url to member's Avatar image"
   avatarUri: String
 
   "Short text chosen by member to share information about themselves"
   about: String
 
-  "Blocknumber when member was registered"
-  registeredAtBlock: Int!
-
   "Member's controller account id"
   controllerAccount: Bytes!
 
   "Member's root account id"
   rootAccount: Bytes!
 
-  happenedIn: Block!
+  "Blocknumber when member was registered"
+  registeredAtBlock: Block!
+
+  "Timestamp when member was registered"
+  registeredAtTime: BigInt!
+
+  "How the member was registered"
+  entry: MembershipEntryMethod!
+
+  "The type of subscription the member has purchased if any."
+  subscription: BigInt
+}
+
+# TODO: add description to fields
+# TODO: revise property requirement for the new and existing fields derived from content-metadata-protobuf
+
+type ChannelCategory @entity {
+  id: ID!
+
+  name: String
+}
+
+type Person @entity {
```

Comment:
Lets not include Person, Series, Season.. anything that will not be part of Sumer release

Reply:
Person, Series, Season, and Playlist were removed.